azure-monitor-opentelemetry-distro/azure/monitor/opentelemetry/distro/__init__.py

```diff
@@ -94,16 +91,7 @@ def configure_azure_monitor(**kwargs) -> None:
 
 
 def _get_resource(configurations: Dict[str, ConfigurationValue]) -> Resource:
```

Comment:
Change ConfigurationValue to include Resource

Reply:
Done